src/wayland/CMakeLists.txt

```diff
@@ -19,6 +24,7 @@ target_link_libraries(mirwayland
 
 target_include_directories(mirwayland
   PUBLIC
+    ${CMAKE_CURRENT_SOURCE_DIR}/base
```

Comment:
Simpler to place files directly in `src/wayland`

Although the header should probably be in include/wayland/mir/wayland (along with adding this directory to install and packaging).

Reply:
Done